test/flux_source_test.go

```diff
@@ -85,7 +85,8 @@ var _ = Describe("Flux source integration", func() {
 		kubectl.Env = append(kubectl.Environ(), fmt.Sprintf("KUBECONFIG=%s", kubeconfig))
 		kubectl.Stdout = os.Stdout
 		kubectl.Stderr = os.Stderr
-		kubectl.Run()
+		err := kubectl.Run()
+		Expect(err).NotTo(HaveOccurred())
```

Comment:
`ExpectWithOffset` would have the marginal advantage of reporting the caller (assumed to be the test itself) as the location of the failure, rather than this helper.